tests/integration/cocoon/test_cocoon_portfolios.py

```diff
@@ -334,3 +336,211 @@ def test_load_from_data_frame_a_portfolios_failure(
                     sub_holding_keys=sub_holding_keys,
                 ),
             )
+
+    @parameterized.expand(
```

Comment:
These tests are great, nice work! Do you think we should also check:

1) That the property is populated with a value on the Portfolio? 
2) A test with multiple properties passed as dictionaries?